VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from '../logging/logger.service';
+
+import { IHearingControlsState, IHearingControlStates, VideoControlCacheService } from './video-control-cache.service';
+
+describe('VideoControlCacheService', () => {
+    let sut: VideoControlCacheService;
+    let loggerSpy: jasmine.SpyObj<LoggerService>;
+
+    beforeEach(() => {
+        loggerSpy = jasmine.createSpyObj<LoggerService>('LoggerService', ['warn', 'info']);
+        sut = new VideoControlCacheService(loggerSpy);
+
+        window.localStorage.clear();
+    });
+
+    it('should load settings from local storage when constructed', () => {
```

Comment:
This doesn't use the sut object, maybe declare that in another describe for all the tests where it is actually used.